inductiva/coastal/README.md

```diff
@@ -1,45 +1,4 @@
-# Coastal dynamics
```

Comment:
where is this going to?

Reply:
No where, for now! This part belongs on the simulators, which we are not casting the most focus now!